packages/vm/lib/state/cache.ts

```diff
@@ -51,30 +51,39 @@ export default class Cache {
     }
   }
 
+  /**
+   * Returns true if the key was deleted and thus existed in the cache earlier
+   * @param key - trie key to lookup
+   */
+  keyIsDeleted(key: Buffer): boolean {
+    const keyStr = key.toString('hex')
+    const it = this._cache.find(keyStr)
+    if (it.node) {
+      return it.value.deleted
+    }
+    return false
```

Comment:
This is currently not covered by tests:
https://codecov.io/gh/ethereumjs/ethereumjs-vm/compare/2009f18c81f2fe253e846333ace57eb77625dd60...681418a97f5e27b56da0f5ca63a6fd4acb244c2b/src/packages/vm/lib/state/cache.ts?before=packages/vm/lib/state/cache.ts#L64

Reply:
Added test for this